controller.js

```diff
@@ -48,6 +49,15 @@ exports.getAllArticles = (req, res, next)=>{
     .catch(next)
 }
 
+exports.patchArticleVote = (req, res, next)=>{
+    const article_id = req.params.article_id
+    Promise.all([checkArticleExists(article_id),updateArticleVote(req.params.article_id,req.body.inc_votes)])
+    .then((body)=>{
+        updatedArticle = body[1]
+        res.status(200).send(updatedArticle)
```

Comment:
As we are only sending back one piece of data, we should be only sending back an object rather than an array here. Also we should always send our data back on an object like we've done before. So an object as the value on a key of something descriptive. You may need to update your tests to get this implemented, make sure they fail first!